src/app/integrations/intacct/intacct.fixture.ts

```
@@ -1,7 +1,11 @@
 import { minimalUser } from "src/app/core/interceptor/jwt.fixture";
+import { AccountingExportSummary } from "src/app/core/models/db/accounting-export-summary.model";
+import { Error } from "src/app/core/models/db/error.model";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)